front/style/pages/_referent.scss

```diff
@@ -126,56 +307,150 @@
         }
     }
 
-    & nav.referent-menu,
-    & div.referent-sub-menu {
-        text-align: center;
-        margin-bottom: 20px;
+    // updated
 
-        ul {
-            display: inline-block;
-            border-bottom: 1px solid $gray--light;
+    .datagrid__pre-table {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        margin-bottom: 40px;
 
-            li {
-                display: inline-block;
-                margin: 0 20px;
+        h3 {
+            font-size: 24px;
```

Comment:
Can't we use existing mixins for typography (font-size and line-height)?

Reply:
well all existing ones are not fitting with new designs